app/SetmoreAppointment.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class SetmoreAppointment extends Model {
+    protected $table = 'setmore';
+    public $timestamps = false;
+    protected $fillable = ['id', 'setmore_key', 'start_time', 'duration', 'staff_key', 'staff_name', 'service_key', 'service_description', 'customer_cid', 'created_at'];
```

Comment:
I think we maybe found that the fillable is unnecessary?

Reply:
According to the docs it is still required... https://laravel.com/docs/10.x/eloquent

Reply:
I'm pretty sure that's only required if we are using batch creations (I've never really been sure how this works, but I think I saw something about that somewhere). The reason would be that you need to tell it which fields to look for in those creations. However, how we are creating objects everywhere, I don't think it's necessary

Reply:
Yeah if you look here, it's the `create()` method that requires the fillable. So we only need the fillable on objects that we're using create on https://laravel.com/docs/10.x/eloquent#inserts

Reply:
Removed. Thank you